### PR TITLE
Header and default style

### PR DESCRIPTION
**Pull request by NoSloppy**

### CODE REVIEW

File: style_editor.html [hunk 1]
```diff
@@ -1,4 +1,12 @@
+<!DOCTYPE html>
+
+<html lang="en">
```
Comment (profezzorn) [inline style_editor.html:3]: Shouldn't there also be a </html> tag at the end?

File: style_editor.html [hunk 2]
```diff
@@ -1,4 +1,12 @@
...
 <head>
+ <title>Style Editor</title>
```
Comment (profezzorn) [inline style_editor.html:5]: Style Editor -> ProffieOS Style Editor

File: style_editor.html [hunk 3]
```diff
@@ -1,4 +1,12 @@
...
+  <meta name="description" content="ProffieOS Style Editor">
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (profezzorn) [inline style_editor.html:8]: What does this do?

File: style_editor.html [hunk 4]
```diff
@@ -1,4 +1,12 @@
+<!DOCTYPE html>
+
+<html lang="en">
 <head>
+ <title>Style Editor</title>
+  <meta name="description" content="ProffieOS Style Editor">
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
```
Comment (profezzorn) [inline style_editor.html:7]: This seems redundant, why do we need it?

File: style_editor.html [hunk 5]
```diff
@@ -982,7 +990,7 @@
 
   var str = new URL(window.location.href).searchParams.get("S");
   if (!str) {
-    str = "InOutHelper<SimpleClash<Lockup<Blast<Blue,White>,AudioFlicker<Blue,White>>,White>, 300, 800>";
+    str = "Layers<Red,ResponsiveLockupL<White,TrInstant,TrInstant,Int<26000>>,ResponsiveLightningBlockL<White>,ResponsiveMeltL<Mix<TwistAngle<>,Red,Yellow>>,ResponsiveDragL<White>,ResponsiveClashL<White,TrInstant,TrFade<200>,Int<26000>>,ResponsiveBlastL<White>,ResponsiveBlastWaveL<White>,ResponsiveBlastFadeL<White>,ResponsiveStabL<White>,InOutTrL<TrWipe<300>,TrWipeIn<500>>>";
```
Comment (profezzorn) [inline style_editor.html:993]: Unless this get automatically formatted into one line per layer at some point later, I think we should have the default be formatted properly.

File: style_editor.html (hunk 3 above)
Comment (NoSloppy) [reply style_editor.html]: This should ensure it looks right on mobile devices.

File: style_editor.html (hunk 4 above)
Comment (NoSloppy) [reply style_editor.html]: Edited. New version has these lines:
<meta http-equiv="X-UA-Compatible" content="IE=edge">
This should ensure it's not rendered as an older version of Internet Explorer, which tends to be buggy.

  <meta name="description" content="ProffieOS Style Editor: Customize your Proffieboard saber blade animations with this browser based tool.">
This should show up in search engine results.

File: style_editor.html (hunk 2 above)
Comment (NoSloppy) [reply style_editor.html]: yes.

File: style_editor.html (hunk 1 above)
Comment (NoSloppy) [reply style_editor.html:3]: Yes, someone ate it, fixed.

File: style_editor.html (hunk 2 above)
Comment (NoSloppy) [reply style_editor.html]: yes.

File: style_editor.html (hunk 4 above)
Comment (NoSloppy) [reply style_editor.html]: Edited. New version has these lines:
<meta http-equiv="X-UA-Compatible" content="IE=edge">
This should ensure it's not rendered as an older version of Internet Explorer, which tends to be buggy.

  <meta name="description" content="ProffieOS Style Editor: Customize your Proffieboard saber blade animations with this browser based tool.">
This should show up in search engine results.

File: style_editor.html (hunk 3 above)
Comment (NoSloppy) [reply style_editor.html]: This should ensure it looks right on mobile devices.

File: style_editor.html (hunk 5 above)
Comment (NoSloppy) [reply style_editor.html:993]: calling DoLayerize() in initGL seems to work.

File: style_editor.html [hunk 6]
```diff
@@ -1,4 +1,12 @@
+<!DOCTYPE html>
+
+<html lang="en">
 <head>
+  <meta charset="UTF-8">
+  <meta name="description" content="ProffieOS Style Editor: Customize your Proffieboard saber blade animations with this browser based tool.">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (profezzorn) [inline style_editor.html:7]: What does this do?

File: style_editor.html (hunk 3 above)
Comment (profezzorn) [reply style_editor.html]: doesn't it also disable zooming?
(Which may make it very hard to use on mobile devices.)

File: style_editor.html (hunk 4 above)
Comment (profezzorn) [reply style_editor.html]: These comments should be in the code, not in the code review.

The X-UA-Compatible thing seems suspicious, I don't think we want or need that. (Unless you have a good reason for it.)

Comment (NoSloppy) [reply style_editor.html]: It's an older code sir, but it checks out. From what I can gather, it seems to be good practice, although a rather dated one at this point.
If it's been fine without it up till now, it's likely fine to 86 it.

File: style_editor.html (hunk 3 above)
Comment (NoSloppy) [reply style_editor.html]: Zooming is fine, however I don't see it making a difference with or without. Again, it's been fine up to now, so there's no need I suppose.

**Pull request merged**